src/opensearch.py

```diff
@@ -40,6 +40,8 @@ class OpenSearchSnap(OpenSearchDistribution):
     _BASE_SNAP_DIR = "/var/snap/opensearch"
     _SNAP_DATA = f"{_BASE_SNAP_DIR}/current"
     _SNAP_COMMON = f"{_BASE_SNAP_DIR}/common"
+    _ETC = f"{_SNAP_DATA}/etc/opensearch"
+    _USR = f"{_SNAP_DATA}/usr/share/opensearch"
```

Comment:
I would perhaps remove these 2 and reference the full path from the `Paths` object, unless we're planning on using them in multiple places?